NGitLab/Impl/PipelineScheduleClient.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using NGitLab.Extensions;
+using NGitLab.Models;
+
+namespace NGitLab.Impl;
+
+internal class PipelineScheduleClient : IPipelineScheduleClient
+{
+    private readonly API _api;
+
+    private readonly string _projectPath;
+    private readonly string _schedulesPath;
+
+    public PipelineScheduleClient(API api, ProjectId projectId)
+    {
+        _api = api;
+        _projectPath = $"{Project.Url}/{projectId.ValueAsUriParameter()}";
```

Comment:
Why have this field? It's only used at construction. You can make it a local variable or inline its value when setting the `_schedulesPath` field.

Reply:
I think I thought I was going to need it at some point in the method, and then forgot to clean that up.

Removed